app/routes/_layout+/kits.$kitId.add-to-existing-booking.tsx

```diff
@@ -256,12 +242,6 @@ export default function ExistingBooking() {
             allowClear
             closeOnSelect
             required={true}
-            transformItem={(item) => ({
```

Comment:
Not sure why we remove this.

Reply:
I removed this because we do not need it at all. We are formatting the dates in `transformItem` but we already get the formatted dates from the server.